chainer/testing/function.py

```diff
@@ -69,6 +69,17 @@ def generate_grad_grad_inputs(self, inputs_template):
             for a in inputs_template])
         return grad_grad_inputs
 
+    def check_forward_outputs(self, expected_outputs, outputs):
```

Comment:
As far as I know, `assert actual == desired` is more common than the opposite.
```suggestion
    def check_forward_outputs(self, outputs, expected_outputs):
```

Reply:
Hmm, I also prefer the `actual, expected` order.
However, I think `expected, actual` is actually more common (perhaps historically, due to [JUnit](http://junit.sourceforge.net/javadoc/org/junit/Assert.html)).
(see also: ["historical note" on googletest primer](https://github.com/google/googletest/blob/master/googletest/docs/primer.md#binary-comparison) )

But we don't necessarily have to follow that convention.

Reply:
`numpy.testing.assert_array_equal` has `actual, expected` order.
I think it's ok to follow that.

Reply:
Python, or `unittest`, was leaving "actual/expected" in favor of "first/second" so I think there's no real consensus but noticed now that NumPy follows `actual, expected` so maybe we can change?

Reply:
> However, I think `expected, actual` is actually more common (perhaps historically, due to [JUnit](http://junit.sourceforge.net/javadoc/org/junit/Assert.html)).

I didn't know this, sorry.:bow:

Reply:
NP, thank you for pointed out.
Changed the order in #6437.